### PR TITLE
add module for dashboardunifie

### PR DESCRIPTION
**Pull request by arnnow**
This is an integration for "dashboardunifié"

This will add a webhook integration

### CODE REVIEW

File: alerting/dashboardunifie/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# ALERTING Dashboard Unifie SignalFx integrations
+
+## How to use this module
+
+```hcl
+module "signalfx-integrations-alerting-dashboardunifie" {
+  source  = "github.com/claranet/terraform-signalfx-integrations.git//alerting/dashboardunifie"
+
+  dashboardunifie_url          = var.dashboardunifie_url
+  header_authorization_base64  = base64encode("username:${data.vault_generic_secret.dashboardunifie.data["token"]}")
```
Comment (xp-1000) [inline alerting/dashboardunifie/README.md:10]: why not using base64encode function inside your module to ease the import for the user?

File: alerting/dashboardunifie/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# ALERTING Dashboard Unifie SignalFx integrations
```
Comment (xp-1000) [inline alerting/dashboardunifie/README.md:1]: I would prefer `UnifiedDashboard` is ok for you?

File: alerting/dashboardunifie/integrations-dashboardunifie.tf [hunk 3]
```diff
@@ -0,0 +1,21 @@
+resource "signalfx_webhook_integration" "sfx_integration" {
+  name    = format("%s", var.suffix)
+  enabled = var.enabled
+  url     = var.dashboardunifie_url
+  headers {
+    header_key   = "Authorization"
+    header_value = "Basic ${var.header_authorization_base64}"
```
Comment (xp-1000) [inline alerting/dashboardunifie/integrations-dashboardunifie.tf:7]: use base64encode here

**Review by xp-1000 — CHANGES_REQUESTED** [review]

File: alerting/dashboardunifie/README.md (hunk 1 above)
Comment (arnnow) [reply alerting/dashboardunifie/README.md]: Yes, i pushed a change so this is managed on the module side & updated the readme

File: alerting/dashboardunifie/README.md (hunk 2 above)
Comment (arnnow) [reply alerting/dashboardunifie/README.md]: Changed

File: alerting/dashboardunifie/integrations-dashboardunifie.tf (hunk 3 above)
Comment (arnnow) [reply alerting/dashboardunifie/integrations-dashboardunifie.tf]: Changed

**Review by xp-1000 — APPROVED** [review]
LGTM thanks

**Pull request merged**